src/plugins/kibana/public/settings/sections/kibi_relations/controllers/relations.js

```diff
@@ -26,7 +26,7 @@ define(function (require) {
             scope.$apply(function () {
               ngModelCtrl.$setViewValue(elm.val());
             });
-          }, attr.ngDebounce || 1000);
+          }, attr.ngDebounce || 1);
```

Comment:
Hi thanks for the contribution
Here the issue was that the time to debounced was defined here
https://github.com/sirensolutions/kibi/blob/master/src/plugins/kibana/public/settings/sections/kibi_relations/index.html#L186
but not taken as the name of the attribute should be kibiDebounce instead of ngDebounce
Could you please change it to 
```
}, attr.kibiDebounce || 1000);
```
And we will be happy to accept the PR